db/migrate/20160315014311_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration
+  def change
+    create_table :posts do |t|
+      t.string :post_title
+      t.string :post_datestamp
+      t.string :post_text
```

Comment:
Since this is a `posts` table, do you need to indicate `post_` in attribute names?